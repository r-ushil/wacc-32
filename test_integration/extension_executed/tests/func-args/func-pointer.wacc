begin
    int bar(int x) is 
        int y = x + 1;
        return y
    end

    int foo(int(int) b, int w) is
        int z = call b(w);
        return z
    end

    int res = call foo(bar, 3);
    exit res
end
