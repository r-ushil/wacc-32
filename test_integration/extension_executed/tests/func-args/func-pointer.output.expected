--------- stdout ---------
--------------------------
Exit code: 4
